homeassistant/components/zwave_js/api.py

```diff
@@ -287,13 +287,22 @@ def websocket_get_configuration_values(
     values = node.get_configuration_values()
     result = {}
     for value in values:
+        metadata = values[value].metadata
         result[value] = {
-            "property": values[value].data["property"],
-            "property_name": values[value].data["propertyName"],
-            "metadata": values[value].data["metadata"],
-            "value": values[value].data.get(
-                "value"
-            ),  # Some config properties don't return a value
+            "property": values[value].property_,
+            "property_name": values[value].property_name,
+            "property_key": values[value].property_key,
+            "type": values[value].type.value,
```

Comment:
now that the upstream PR has been accepted, this should change from `type` to `configuration_value_type`